plugins/MauticFocusBundle/Translations/en_US/messages.ini

```
@@ -1,112 +1,126 @@
-mautic.focus.form.when = "When to engage?"
-mautic.focus.graph.clicks = "Clicks"
-mautic.focus.graph.stats = "Focus Stats"
-mautic.focus.graph.submissions = "Submissions"
-mautic.focus.graph.views = "Views"
-mautic.focus.install.description = "Copy this line of code into your site, preferably right before the closing body tag. Alternatively, show this focus item through a campaign action in response to a page view on a page that has the Mautic tracking script."
-mautic.focus.install.header = "Focus Installation"
-mautic.focus.new = "New Focus"
-mautic.focus.noresults.tip = "Get the attention of visitors on your website with Focus Forms, Focus Clicks, or Focus Notifications."
-mautic.focus.style.bar_description = "Display a bar across the top of the page."
-mautic.focus.style.bar = "Bar"
-mautic.focus.style.modal_description = "Display a popup in the middle of the page."
-mautic.focus.style.modal = "Modal"
-mautic.focus.style.notification_description = "Display a small window in a corner of the page."
-mautic.focus.style.notification = "Notification"
-mautic.focus.style.page_description = "Display a window that covers the entire page."
-mautic.focus.style.page = "Full Page"
-mautic.focus.tab.focus_colors = "Colors"
-mautic.focus.tab.focus_content = "Content"
-mautic.focus.tab.focus_style = "Style"
-mautic.focus.tab.focus_type = "Focus"
-mautic.focus.thead.style = "Focus Style"
-mautic.focus.thead.type = "Focus Type"
-mautic.focus.type.form = "Form"
-mautic.focus.type.link = "Link"
-mautic.focus.type.notice = "Notice"
-mautic.focus.website_placeholder = "Enter a website below and I'll grab a snapshot so you can see an example of what your Focus will look like on your site as you build it!"
-mautic.focums.html_mode.tooltip = "Pick one mode. Content will not carry over when switching between modes"
-mautic.focus.campaign.event.show_focus = "Show Focus Item"
-mautic.focus.campaign.event.show_focus_descr = "Trigger Focus Item"
+mautic.focus.form.website.tooltip="Enter your website for Mautic to grab a screenshot and display an example."
+mautic.focus.form.website="Website"
+mautic.focus.form.when.5seconds="5 second delay"
+mautic.focus.form.when.5seconds.description="Show the focus after a 5 second delay"
+mautic.focus.form.when.immediately="Upon arrival"
+mautic.focus.form.when.immediately.description="Show the focus immediately upon arrival"
+mautic.focus.form.when.leave="Visitor intends to leave"
+mautic.focus.form.when.leave.description="Show the focus when the visitor intends to leave"
+mautic.focus.form.when.minute="1 minute delay"
+mautic.focus.form.when.minute.description="Show the focus after a 1 minute delay."
+mautic.focus.form.when.scroll_bottom="After scrolling to bottom"
+mautic.focus.form.when.scroll_bottom.description="Show the focus after scrolling to the bottom."
+mautic.focus.form.when.scroll_middle="After scrolling to middle"
+mautic.focus.form.when.scroll_slight="After slightly scrolling down"
```

```suggestion
mautic.focus.form.when.scroll_middle="After scrolling to middle"
mautic.focus.form.when.scroll_middle.description="Show the focus after the user has scrolled to the middle of the page."
mautic.focus.form.when.scroll_slight="After slightly scrolling down"
mautic.focus.form.when.scroll_slight.description="Show the focus after the user scrolls down slightly on the page."
```
The description is missing.